source/dart/lib/astronomy.dart

```diff
@@ -1,7 +1,24 @@
 library astronomy;
 
+import 'dart:core';
 import 'dart:math';
 
+import 'package:meta/meta.dart';
+
+/// The number of minutes in a day.
+const minutesPerDay = 60.0 * 24.0;
+
+/// The number of seconds in a day.
+const secondsPerDay = 60.0 * minutesPerDay;
+
+/// The number of milliseconds in a day.
+const millisecondsPerDay = 1000.0 * secondsPerDay;
+
+const daysPerTropicalYear = 365.24217;
+const dayPerMillennium = 365250.0;
+
+@visibleForTesting
+@immutable
```

Comment:
These are coming from the package, meta, hosted inside dart lang repository itself https://github.com/dart-lang/sdk/tree/main/pkg/meta as dart doesn't have `internal` concept and if something is supposed not to be exposed to outside it should be simply prefixed with underscore `_` (no private and so)